rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistResponseCode.java

```diff
@@ -3,7 +3,10 @@
 public enum WhitelistResponseCode {
     GENERIC_ERROR(-10),
```

Comment:
as in the other refactors (see [FeePerKbResponseCode](https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/co/rsk/peg/feeperkb/FeePerKbResponseCode.java#L8) and [FederationChangeResponseCode](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/federation/FederationChangeResponseCode.java#L13)), we should add a 
`UNAUTHORIZED_CALLER(-10)` response code to be returned [here](https://github.com/rsksmart/rskj/pull/2460/files#diff-8dee29de1371a107fbe90b96332f520e0f331c1da75eb3e59965f5ed27b64db4R168) and [here](https://github.com/rsksmart/rskj/pull/2460/files#diff-8dee29de1371a107fbe90b96332f520e0f331c1da75eb3e59965f5ed27b64db4R135)

Reply:
Done!